zerver/lib/actions.py

```diff
@@ -2044,6 +2055,29 @@ def get_default_subs(user_profile):
     # to some day further customize how we set up default streams for new users.
     return get_default_streams_for_realm(user_profile.realm)
 
+def make_dict(row):
```

Comment:
this deserves a more unique name, like `make_default_stream_dict`, if we're moving it out here